learning_journal/learning_journal/templates/404.jinja2

```diff
@@ -0,0 +1,8 @@
+{% extends "index.jinja2" %}
+
+{% block content %}
+<div class="content">
+  <h1><span class="font-semi-bold">Pyramid</span> <span class="smaller">Alchemy scaffold</span></h1>
```

Comment:
You probably want to consider updating your 404 template to remove references to the Pyramid world